README.md

```diff
@@ -174,34 +172,36 @@ const { Branch } = odd.path
 
 // List the user's private files
 await fs.ls(
-  odd.path.directory(Branch.Private)
+  odd.path.directory(Branch.Private),
```

Comment:
Is this trailing comma from auto-linting? It's alright in an object, but a bit strange in function params.

In general, my preference is to omit trailing commas even in objects because it's just extra syntactic noise. But I could be convinced that it's alright for objects.

Reply:
It's the formatter yeah, I've just adjusted the formatting so that it doesn't do this.
I've disabled trailing commas for arguments and params.
If it needs to be changed elsewhere let me know, these are all the options: https://dprint.dev/plugins/typescript/config/